README.md

```diff
@@ -82,6 +84,47 @@ or as admin:
 - username: `tech@gisaia.com`
 - password: `admin`
 
+# ARLAS AIAS deployment
+
+The AIAS (ARLAS Item and Asset Services) deployment has:
+- [apisix](https://apisix.apache.org/)
+- [arlas-wui-iam](https://github.com/gisaia/ARLAS-wui-iam)
+- [arlas-iam-server](https://github.com/gisaia/ARLAS-IAM)
+- [postgres](https://www.postgresql.org/)
+- [arlas-wui](https://github.com/gisaia/ARLAS-wui)
+- [arlas-hub](https://github.com/gisaia/ARLAS-wui-hub)
+- [arlas-builder](https://github.com/gisaia/ARLAS-wui-builder)
+- [arlas-persistence-server](https://github.com/gisaia/ARLAS-persistence)
+- [arlas-permissions-server](https://github.com/gisaia/ARLAS-permissions)
+- [arlas-server](https://github.com/gisaia/ARLAS-server)
+- [agate](https://github.com/gisaia/aias)
```

Comment:
aias is private

Reply:
yes but when it will be public, this will be ok.